src/index.js

```diff
@@ -19,7 +19,7 @@ import {
  */
 export function execute(
   sitekey,
-  {locale = 'en', position = 'bottomright'} = {}
+  {locale = 'en', position = 'bottomright', baseUrl = 'www.google.com'} = {}
```

Comment:
```suggestion
  {locale = 'en', position = 'bottomright', baseUrl = 'https://www.google.com'} = {}
```
@supriyo-biswas In order to be more flexible, I think would be nice if we also include the protocol in the url. WDYT?

Reply:
Thanks for the suggestion, I've committed the suggested changes.